app/models/survey.rb

```diff
@@ -7,11 +7,14 @@
 #  description                          :text
 #  created_at                           :datetime         not null
 #  updated_at                           :datetime         not null
+#  first_question_id                    :integer
 #
 
 class Survey < ActiveRecord::Base
   has_many :question_orders
   has_many :survey_responses
 
-  validates :description, :name, presence: true
+  belongs_to :first_question, foreign_key: :first_question_id, class_name: 'Question'
+
+  validates :description, :name, :first_question_id, presence: true
```

Comment:
I don't think a `first_question_id` needs to be present for a valid survey.

Example scenarios:
- User creates survey, doesn't add first question
- User creates survey with survey title and description, but doesn't add first question

They should be able to save their surveys in the above scenarios.